Shell/Property.cpp

```diff
@@ -687,6 +684,12 @@ void Property::scan(TermList ts,bool unit,bool goal)
         break;
     }
   } else {
+    if(t->isSort()){
+      if(t->arity() > _maxTypeConArity)
+      _maxTypeConArity = t->arity();
```

Comment:
Indentation here is confusing. Assume L689 wants indenting by one and then we `return` in any event if `t->isSort()`.

Reply:
Thanks, resolved